Tasks/NuGetInstallerV0/nugetinstaller.ts

```diff
@@ -189,6 +189,7 @@ async function main(): Promise<void> {
         }
 
         tl.setResult(tl.TaskResult.Succeeded, tl.loc("PackagesInstalledSuccessfully"));
```

Comment:
Why are we setting success result (and emitting the success message?) then throwing. Let's just set the result to failed and emit the loc string

Reply:
Changed this!

Reply:
Now we're using a feature flag. I included the success message for when it's on